Library/Formula/ffmpeg.rb

```diff
@@ -33,6 +33,7 @@ class Ffmpeg < Formula
   option "with-libsoxr", "Enable the soxr resample library"
   option "with-webp", "Enable using libwebp to encode WEBP images"
   option "with-zeromq", "Enable using libzeromq to receive commands sent through a libzeromq client"
+  option "with-libbs2b", "Enable using libbs2b"
```

Comment:
Please remove the line. With it `brew info ffmpeg` shows this:

```
…snip…
--with-libbluray
	Build with libbluray support
--with-libbs2b
	Enable using libbs2b
--with-libcaca
	Build with libcaca support
--with-libquvi
	Build with libquvi support
…snip…
```

Without it:

```
…snip…
--with-libbluray
	Build with libbluray support
--with-libbs2b
	Build with libbs2b support
--with-libcaca
	Build with libcaca support
--with-libquvi
	Build with libquvi support
…snip…
```

I thus don’t think it’s necessary.